### ISSUE TITLE
The color of $ is different from atom for bash

### ISSUE THREAD

**Issue by xdc0209** [akamud/vscode-theme-onedark#74]
Filename
-----------------------------
passwod.sh

Code
-----------------------------
```bash
#!/bin/bash
echo "Try with [ $ip something else ... $user something else ... $password ]."   
```

Describe
-----------------------------
 the var in quote is not same with atom one dark. the color of $ is different.

## Atom Original theme Screenshot
![image](https://user-images.githubusercontent.com/3997527/30779333-352b0a04-a120-11e7-8bd4-002c1f2ee73b.png)

## VSCode theme Screenshot
![image](https://user-images.githubusercontent.com/3997527/30779292-69b26106-a11f-11e7-9225-c947bd8d0051.png)

## Versions used

* VSCode version: Code 1.16.1 (27492b6bf3acb0775d82d2f87b25a93490673c6d, 2017-09-14T16:38:23.027Z)
* Theme version: Version 2.0.4
* Atom version: 1.20.1 x64

## Extra information

[Any other information that might be useful]  
[Are you using a Plugin? Some languages (like Rust) require a plugin from the Marketplace to work on VSCode, please tell us if you are using any plugins that can be affecting the way VSCode renders your code]

**Comment by akamud** [comment]
Thanks for letting me know. This will be fixed in 2.0.5 which will be released later this week.

**Comment by akamud** [comment]
Release 2.0.5 is available.
